### PR TITLE
boards/same54-xpro: configure pins for UART HW flow control & ethernet LED

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

On EXT1 the pins for RTS and CTS are accessible, so configure them.

### CODE REVIEW

File: boards/same54-xpro/include/periph_conf.h [hunk 1]
```diff
@@ -115,10 +115,6 @@ static const uart_conf_t uart_config[] = {
         .dev      = &SERCOM2->USART,
         .rx_pin   = GPIO_PIN(PB, 24),
         .tx_pin   = GPIO_PIN(PB, 25),
-#ifdef MODULE_PERIPH_UART_HW_FC
-        .rts_pin  = GPIO_UNDEF,
-        .cts_pin  = GPIO_UNDEF,
-#endif
```
Comment (dylad) [inline boards/same54-xpro/include/periph_conf.h:121]: Why did you remove those one ?

Comment (benpicco) [reply boards/same54-xpro/include/periph_conf.h:121]: It's not needed, we check `.tx_pad == UART_PAD_TX_0_RTS_2_CTS_3`

**Review by dylad — APPROVED** [review]
ACK.
